src/items/external-blocks.md

```diff
@@ -53,10 +55,19 @@ extern "abi" for<'l1, ..., 'lm> fn(A1, ..., An) -> R`, where `'l1`, ... `'lm`
 are its lifetime parameters, `A1`, ..., `An` are the declared types of its
 parameters and `R` is the declared return type.
 
+## Statics
+
 Statics within external blocks are declared in the same way as statics outside of external blocks,
 except that they do not have an expression initializing their value.
 It is `unsafe` to access a static item declared in an extern block, whether or
-not it's mutable.
+not it's mutable, because there is nothing guaranteeing that the bit pattern at the static's
+memory is valid for the type it is declared with, since some arbitrary (e.g. C) code is in charge
+of initializing the static.
+
+Extern statics can be either immutable or mutable just like statics outside of external blocks.
+An immutable static *must* be initialized before any Rust code is executed. It is not enough for
+the static to be initialized before Rust code reads from it. This may change in the future, but for
+now, if you want to lazily initialize such a static, use a mutable static.
```

Comment:
Remove the last sentence. The reference is not a user guide / should not contain advice.